examples/stable/test/end_to_end/candid_rpc/workspaces/package.json

```diff
@@ -0,0 +1,9 @@
+{
+    "dependencies": {
+        "azle": "0.30.0"
+    },
+    "scripts": {
+        "pretest": "tsx test/pretest.ts",
+        "test": "AZLE_CHECK_GLOBAL_STATE=false jest"
+    }
+}
```

Comment:
Trying to start a review

Reply:
Yeah I like this comments much better, the behave more normally in this extension